packages/datadog-plugin-oracledb/test/index.spec.js

```diff
@@ -295,18 +343,20 @@ describe('Plugin', () => {
             }
           )
 
-          it('should set the service name', done => {
-            agent.assertSomeTraces(traces => {
-              expect(traces[0][0]).to.have.property('name', expectedSchema.outbound.opName)
-              expect(traces[0][0]).to.have.property('service', 'custom')
-            }).then(done, done)
-            connection.execute(dbQuery)
+          it('should set the service name', async () => {
```

Comment:
I think the test case here is reversed with the above? Meaning that it says "with service undefined" but then defines a service string and vice versa.